### PR TITLE
Updated module to use new environment label value

### PR DESCRIPTION
**Pull request by tthayer**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -11,14 +11,15 @@ locals {
...
 module "label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.15.0"
-  namespace  = var.namespace
-  stage      = var.stage
-  name       = var.name
-  delimiter  = var.delimiter
-  attributes = compact(concat(var.attributes, ["workers"]))
-  tags       = local.tags
-  enabled    = var.enabled
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref = tags/0.16.0"
```
Comment (maximmi) [inline main.tf:14]:
```suggestion
  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
```

**Comment by tthayer** [comment]
Thanks for the review! That space around the equals thing has been plaguing me for a while.

**Comment by aknysh** [comment]
@tthayer 

```
TestExamplesComplete 2020-04-29T21:42:03Z command.go:121: Error: Unsupported argument                                                                                                    
TestExamplesComplete 2020-04-29T21:42:03Z command.go:121:                                                                                                                                
TestExamplesComplete 2020-04-29T21:42:03Z command.go:121:   on ../../main.tf line 178, in module "autoscale_group":                                                                      
TestExamplesComplete 2020-04-29T21:42:03Z command.go:121:  178:   environment = var.environment                                                                                          
TestExamplesComplete 2020-04-29T21:42:03Z command.go:121:                                                                                                                                
TestExamplesComplete 2020-04-29T21:42:03Z command.go:121: An argument named "environment" is not expected here.                                                                          
T
```

File: docs/terraform.md [hunk 2]
```diff
@@ -1,104 +1 @@
...
-| autoscaling_group_name | The AutoScaling Group name |
-| launch_template_arn | ARN of the launch template |
-| launch_template_id | The ID of the launch template |
-| security_group_arn | ARN of the worker nodes Security Group |
-| security_group_id | ID of the worker nodes Security Group |
-| security_group_name | Name of the worker nodes Security Group |
-| workers_role_arn | ARN of the worker nodes IAM role |
-| workers_role_name | Name of the worker nodes IAM role |
-
+Error: template: inputs:10:118: executing "inputs" at <.GetValue>: error calling GetValue: json: error calling MarshalJSON for type types.String: invalid character '.' in string escape code
```
Comment (osterman) [inline docs/terraform.md:1]: @tthayer please have a look at the inputs. Something is malformed causing this error.

Comment (osterman) [reply docs/terraform.md:1]: (it's also possible that it's an existing input you didn't change - we upgraded `terraform-docs` which might lead to that)

**Comment by osterman** [comment]
@tthayer bump

**Pull request closed without merging**